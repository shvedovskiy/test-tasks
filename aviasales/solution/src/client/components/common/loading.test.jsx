/* eslint-disable no-undef */
import React from 'react';

import Loading from './loading';


describe('<Loading />', () => {
  const loading = shallow(<Loading />);

  it('renders properly', () => {
    expect(loading).toMatchSnapshot();
  });
});
