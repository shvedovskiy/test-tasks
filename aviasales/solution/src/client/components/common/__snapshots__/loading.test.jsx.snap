// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loading /> renders properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <_default />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": <h2>
        Подождите минутку, мы ищем билеты...
      </h2>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Подождите минутку, мы ищем билеты...",
      },
      "ref": null,
      "rendered": "Подождите минутку, мы ищем билеты...",
      "type": "h2",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <h2>
          Подождите минутку, мы ищем билеты...
        </h2>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Подождите минутку, мы ищем билеты...",
        },
        "ref": null,
        "rendered": "Подождите минутку, мы ищем билеты...",
        "type": "h2",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
